### PR TITLE
Remove fixed IBC drop name

### PR DESCRIPTION
**Pull request by tmat**
This will cause the build to pick up the latest.

### CODE REVIEW

**Comment by tmat** [comment]
@jasonmalinowski FYI: This change shouldn't flow to master.

**Comment by jasonmalinowski** [comment]
@tmat Is there something we can do inside of the yaml to condition it based on branch name? Git gives no way to stop this to flow short of Tiger dealing with this manually. Tagging @RikkiGibson and his temporary backup @agocke.

**Comment by tmat** [comment]
@jasonmalinowski Yes, that sounds better. 

@jmarolf Should we also make this conditional on branch being built?
```
VisualStudio.ChannelName: 'int.d16.0stg'
```

**Comment by tmat** [comment]
@jasonmalinowski Let's use this PR now to get RPS tests. Meanwhile I can prepare another change that selects the branch conditionally.

File: azure-pipelines-official.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
               -officialSkipTests $(SkipTests)
               -officialSkipApplyOptimizationData $(SkipApplyOptimizationData)
               /p:RepositoryName=$(Build.Repository.Name)
-              /p:VisualStudioIbcSourceBranchName="master-vs-deps"
+              /p:VisualStudioIbcSourceBranchName="dev16.0-vs-deps"
```
Comment (jaredpar) [inline azure-pipelines-official.yml]: Won't this evetnually break the YAML file in master when it flows back?

**Pull request merged**